ScicraftLearn/Minelabs
# Test chemical reactions

Radius property can be adjusted in the Molecules.java file in the "potion" directory. The white and blacklist can be added as arguments to the constructor of a reaction.

To test the features:
- Change the radius
- Add blocks to white or blacklists, see PH3 as example (with bedrock in the blacklist)
- HNO3 is poisonous, H2O extinguishive and NCl3 flammable when stepped upon. These are just POCs and will be made chemically correct in the next sprint
